src/TTextEdit.cpp

```diff
@@ -993,13 +994,49 @@ void TTextEdit::mouseMoveEvent( QMouseEvent * event )
         }
     }
     x = ( event->x() / mFontWidth ) - timeOffset;
-    if( ( x < 0 ) || ( y < 0 ) || ( y > (int) mpBuffer->size()-1 ) )
+    if( ( y < 0 ) || ( y > (int) mpBuffer->size()-1 ) )
```

Comment:
Can `y` never be less than zero now - or is it handled differently (but correctly) if it is...?